packages/embed/src/lib/embed-messages.test.ts

```diff
@@ -1,23 +1,24 @@
-import { TTSService } from '@humeai/voice';
 import { describe, expect, it } from 'vitest';
 
 import {
   parseClientToFrameAction,
   UPDATE_CONFIG_ACTION,
 } from './embed-messages';
 
+export enum TTSService {
+  /** Hume's Text-To-Speech */
+  DEFAULT = 'hume_ai',
+  /** ElevenLab's Text-To-Speech */
+  ELEVEN_LABS = 'eleven_labs',
+  /** Play HT's Text-To-Speech */
+  PLAY_HT = 'play_ht',
+}
+
 describe('parseEmbedMessage', () => {
   it('should parse embed message', async () => {
     const test = () => {
       const action = UPDATE_CONFIG_ACTION({
-        auth: {
-          type: 'apiKey',
-          value: '',
-        },
-        hostname: '',
```

Comment:
meeting note: `hostname`, `reconnectAttempts`, and `debug` are all options we want to continue exposing from our client. Deep will update the Fern generator to support these based on the functionality we already get from [react-use-websocket](https://www.npmjs.com/package/react-use-websocket).

Reply:
These are now supported!